model/ftn/w3ounfmetamd.ftn

```diff
@@ -0,0 +1,2923 @@
+      META(1)%VARNS='expected maximum sea surface elevation (nonlinear,2nd order)'
+      META(1)%VARNG='expected maximum sea surface elevation (nonlinear,2nd order)'
+      META(1)%VMIN = 0
+      META(1)%VMAX = 64
+      !META(1)%VMAX = 32000
+! IFI=2, IFJ=12
+      META => GROUP(2)%FIELD(12)%META
+      META(1)%FSC    = 0.002
+      META(1)%UNITS  = 'm'
+      META(1)%ENAME  = '.mxes'
+      META(1)%VARNM='stmaxd'
+      META(1)%VARNL='standard deviation of maximum sea surface elevation (nonlinear,2nd order)'
+      META(1)%VARNS='std of expected maximum sea surface elevation (nonlinear,2nd order)'
+      META(1)%VARNG='standard deviation of maximum sea surface elevation (nonlinear,2nd order)'
+      META(1)%VMIN = 0
+      META(1)%VMAX = 64
+      !META(1)%VMAX = 32000
+! IFI=2, IFJ=13
+      META => GROUP(2)%FIELD(13)%META
+      META(1)%FSC    = 0.002
+      META(1)%UNITS  = 'm'
+      META(1)%ENAME  = '.mxh'
+      META(1)%VARNM='hmaxe'
+      META(1)%VARNL='expected maximum wave height (linear, 1st order)'
+      META(1)%VARNS='expected maximum wave height (linear, 1st order)'
+      META(1)%VARNG='expected maximum wave height (linear, 1st order)'
+      META(1)%VMIN = 0
+      META(1)%VMAX = 64
+      !META(1)%VMAX = 32000
+! IFI=2, IFJ=14
+      META => GROUP(2)%FIELD(14)%META
+      META(1)%FSC    = 0.002
+      META(1)%UNITS  = 'm'
+      META(1)%ENAME  = '.mxhc'
+      META(1)%VARNM='hcmaxe'
+      META(1)%VARNL='expected maximum wave height from crest (linear, 1st order)'
+      META(1)%VARNS='expected maximum wave height from crest (linear, 1st order)'
+      META(1)%VARNG='expected maximum wave height from crest (linear, 1st order)'
+      META(1)%VMIN = 0
+      META(1)%VMAX = 64
+      !META(1)%VMAX = 32000
+! IFI=2, IFJ=15
+      META => GROUP(2)%FIELD(15)%META
+      META(1)%FSC    = 0.002
+      META(1)%UNITS  = 'm'
+      META(1)%ENAME  = '.sdmh'
+      META(1)%VARNM='hmaxd'
+      META(1)%VARNL='STD of maximum wave height (linear, 1st order)'
+      META(1)%VARNS='STD of maximum wave height (linear, 1st order)'
+      META(1)%VARNG='STD of maximum wave height (linear, 1st order)'
+      META(1)%VMIN = 0
+      META(1)%VMAX = 64
+      !META(1)%VMAX = 32000
+! IFI=2, IFJ=16
+      META => GROUP(2)%FIELD(16)%META
+      META(1)%FSC    = 0.002
+      META(1)%UNITS  = 'm'
+      META(1)%ENAME  = '.sdmhc'
+      META(1)%VARNM='hcmaxd'
+      META(1)%VARNL='STD of maximum wave height from crest (linear, 1st order)'
+      META(1)%VARNS='STD of maximum wave height from crest (linear, 1st order)'
+      META(1)%VARNG='STD of maximum wave height from crest (linear, 1st order)'
+      META(1)%VMIN = 0
+      META(1)%VMAX = 64
+      !META(1)%VMAX = 32000
+! IFI=2, IFJ=17
+      META => GROUP(2)%FIELD(17)%META
+      META(1)%FSC    = 0.001
+      META(1)%UNITS  = '1'
+      META(1)%ENAME  = '.wbt'
+      META(1)%VARNM='wbt'
+      META(1)%VARNL='domiant wave breaking probability'
```

Comment:
"domiant" should be "dominant"?

Reply:
Yes - fixed (not one of my typos this time!)